test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
Any reason not to use `git submodule`? It may make it easier to update if there is a change in toolkit that requires a change in `googleapis`.